pyerrors/fits.py

```diff
@@ -462,6 +462,17 @@ def _standard_fit(x, y, func, silent=False, **kwargs):
 
     if kwargs.get('correlated_fit') is True:
         corr = covariance(y, correlation=True, **kwargs)
+        
+        
+        if kwargs['cutoff']:
```

Comment:
This functionality is not explained in the docstring of ```least_squares```. Please add an explanation such that users are aware of the feature.